lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -10627,7 +10627,7 @@ using namespace Js;
     Var JavascriptOperators::ToNumber(Var aRight, ScriptContext* scriptContext)
```

Comment:
Is this the abstract operation [ToNumber](https://tc39.github.io/proposal-bigint/#sec-tonumber) as defined in the spec? If so it should throw a TypeError if called on a BigInt.

Reply:
removed